auction-server/src/auction/service/mod.rs

```diff
@@ -272,6 +272,124 @@ pub enum ServiceEnum {
     Svm(Service<Svm>),
 }
 
+#[cfg(test)]
+pub use mock_service::MockServiceInner as StatefulMockAuctionService;
```

Comment:
any objection to keep MockServiceInner as the name instead of the alias StatefulMockAuctionService